### DISCUSSION TITLE
Donation box

### DISCUSSION THREAD

**Discussion by ooker777** [pcottle/learnGitBranching#969] [General]
Thanks for the journey. What can I do to show my gratitude about this?

**Comment by pcottle** [comment]
Your kind words of appreciation are more than enough! I have a full time job so I can support this as a passion project. I also appreciate everyone else who pitches in with pull requests and translations and new features, so it's a big collective effort at the end of the day 😇

**Reply by ooker777** [reply]
do you think that if you are free to commit to the project, it will be much better for everyone?

**Reply by pcottle** [reply]
I'm not sure if it needs someone working on it full-time, it's in a pretty stable place.

**Reply by ooker777** [reply]
well, you can contribute to other wonderful projects that you like? :D

**Comment by pcottle** [comment]
I kicked off an application to add a sponsorship button, so stay tuned! But again, don't feel obligated to support financially :)

**Comment by pcottle** [comment]
Sponsorship is approved! Should be a button at the top of the project you can click 😇
